File: HB/pysrc/프로그래머스/레벨0/Day18/문자열안에_문자열.py
def solution(str1, str2):
    answer = 2
    
    for i in range(len(str1)-len(str2)+1):
        a=len(str2)
        for j in range(len(str2)):
            if str1[i+j]==str2[j]:
                a-=1
            else:
                break
            if a==0:
                answer =1
    return answer

File: SB/src/프로그래머스/레벨0/Day10/문자열안에_문자열.java
package 프로그래머스.레벨0.Day10;

public class 문자열안에_문자열 {
    class Solution {
        public int solution(String str1, String str2) {
            return str1.contains(str2) ? 1 : 2;
        }
    }
}
